charts/gardener/controlplane/charts/runtime/templates/admission-controller/configmap-componentconfig.yaml

```diff
@@ -37,6 +37,12 @@ data:
         tls:
           serverCertDir: /etc/gardener-admission-controller/srv
       {{- if .Values.global.admission.config.server.resourceAdmissionConfiguration }}
+      healthProbes:
+        bindAddress: {{ required ".Values.global.admission.config.server.healthProbes.bindAddress is required" .Values.global.admission.config.server.healthProbes.bindAddress }}
+        port: {{ required ".Values.global.admission.config.server.healthProbes.port is required" .Values.global.admission.config.server.healthProbes.port }}
+      metrics:
+        bindAddress: {{ required ".Values.global.admission.config.server.metrics.bindAddress is required" .Values.global.admission.config.server.metrics.bindAddress }}
+        port: {{ required ".Values.global.admission.config.server.metrics.port is required" .Values.global.admission.config.server.metrics.port }}
```

Comment:
Why is this in the `{{- if .Values.global.admission.config.server.resourceAdmissionConfiguration }}` block?

Reply:
Oops, nice catch!